runner.yml.dist

```diff
@@ -0,0 +1,43 @@
+drupal:
+  root: "build"
+  base_url: "http://web:8080/build"
+  site:
+    name: "OpenEuropa"
+    profile: "minimal"
+  database:
+    host: "mysql"
+    port: "3306"
+    name: "oe_time_caching"
+    user: "root"
+    password: ""
+  post_install:
+    # Prepare the instance.
+    - "./vendor/bin/drush en toolbar -y"
+    - "./vendor/bin/drush theme:enable bartik -y"
+    - "./vendor/bin/drush theme:enable seven -y"
+    - "./vendor/bin/drush config-set system.theme default bartik -y"
+    - "./vendor/bin/drush config-set system.theme admin seven -y"
+    - "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y"
+    # Enable the modules.
+    - "./vendor/bin/drush en config_devel -y"
+    - "./vendor/bin/drush en oe_time_caching -y"
+    - "./vendor/bin/drush cr"
+  settings:
+    settings:
+      file_scan_ignore_directories:
+        - "node_modules"
+        - "bower_components"
+        - "vendor"
+        - "${drupal.root}"
+
+commands:
+  drupal:site-setup:
+    - { task: "symlink", from: "../../..", to: "${drupal.root}/modules/custom/oe_time_caching" }
+    - { task: "run", command: "drupal:drush-setup" }
+    - { task: "run", command: "drupal:settings-setup" }
+    - { task: "run", command: "setup:phpunit" }
+    - { task: "run", command: "setup:behat" }
+  setup:phpunit:
+    - { task: "process", source: "phpunit.xml.dist", destination: "phpunit.xml" }
```

Comment:
Let's add also this to the phpunit setup task:

```
    # Generate settings.testing.php, it will be used when running functional tests.
    - { task: "process-php", type: "write", config: "drupal.settings", source: "${drupal.root}/sites/default/default.settings.php", destination: "${drupal.root}/sites/default/settings.testing.php", override: true }
```